### PR TITLE
[OKU-14] Allow creating a Review in BE

### PR DESCRIPTION
**Pull request by enricodelarosa**

### CODE REVIEW

File: controllers/gameReviewsController.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { RequestHandler } from "express";
+
+import { PrismaClient } from "@prisma/client";
+import { create } from "domain";
+const prisma = new PrismaClient();
+
+export const addReview: RequestHandler = async (req, res) => {
```
Comment (idiglove) [inline controllers/gameReviewsController.ts]: add req and res types

File: controllers/gameReviewsController.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        data: {
+            content: content,
+            username: username,
+            gameId: gameId,
+        },
+    });
+
+    if (createdReview) {
+        return res.status(201).send({
+            heading: `Reivew has been created`,
```
Comment (idiglove) [inline controllers/gameReviewsController.ts]:
```suggestion
            heading: `Review has been created`,
```

**Review by idiglove — CHANGES_REQUESTED** [review]

File: controllers/gameReviewsController.ts (hunk 1 above)
Comment (enricodelarosa) [reply controllers/gameReviewsController.ts]: Types added. Will not miss types for req res again.

File: controllers/gameReviewsController.ts (hunk 2 above)
Comment (enricodelarosa) [reply controllers/gameReviewsController.ts]: Spelling corrected

**Review by idiglove — APPROVED** [review]
nice 🏄🏽

**Pull request merged**